### PR TITLE
Fixes Romerol--Comments it Out Until Refactor

### PR DESCRIPTION
**Pull request by Fox-McCloud**
This fixes+updates Romerol, but also comments it out.

Needless to say, there's really no way for this to properly function on our code until we engage in a bit of a species refactor.

At the moment, when you change species, it deletes all your internal and external organs and recreates them.

The way it was done in the PR is NOT the way to go about it; you still wind up with wonky behavior of organs not in lists but still associate with a mob (bad).

There's not an easy way to fix this, and I don't want it runtiming all over the place.

I'd like to reach out to @KasparoVy for some help with the organs thing; he was instrumental in getting species changing to retain damage and prevent reviving from deleting augments and stuff. If it's not too much to ask, could ya look into having `set_species` not remove some types of internal organs? Basically anything outside of the species own organs should be deleted, but anything added should stay.

:cl: Fox McCloud
del: comments out romerol (much like zombies, it will rise again, don't worry!)
tweak: zombies are high and lower pressure immune in addition to immune to heat and tox damage
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
On the case! Thanks for the ping

**Comment by Fox-McCloud** [comment]
@KasparoVy 

Basically, if it's outside of the `has_organ` or `has_limbs` lists, then the organ shouldn't be removed when you change species; ideally, augmented limbs would remain augmented, and prosthetics would also remain.

**Comment by KasparoVy** [comment]
Thanks for clearing up the specifications

File: code/modules/reagents/chemistry/reagents/toxins.dm [hunk 1]
```diff
@@ -221,16 +227,22 @@ datum/reagent/romerol
...
+		var/obj/item/organ/internal/zombie_infection/nodamage/ZI = new()
+		ZI.insert(H)
 	..()
 
-/datum/reagent/stable_mutagen/on_tick()
-	var/datum/reagent/blood/B = locate() in holder.reagent_list
-	if(B && islist(B.data) && !data)
-		data = B.data.Copy()
-	..()
+/datum/reagent/romerol/stable_mutagen/on_mob_life(mob/living/M)
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/toxins.dm]: Fox you absolutely donk, what did you do.

File: code/modules/zombie/organs.dm [hunk 2]
```diff
@@ -4,90 +4,96 @@
 	parent_organ = "head"
 	slot = "zombie_infection"
 	icon_state = "blacktumor"
-	var/converts_living = FALSE
 	var/causes_damage = TRUE
 	var/datum/species/old_species = /datum/species/human
-	var/living_transformation_time = 3
+	var/living_transformation_time = 30
```
Comment (Couls) [inline code/modules/zombie/organs.dm]: ruh roh

File: code/modules/zombie/organs.dm [hunk 3]
```diff
@@ -4,90 +4,96 @@
...
+	if(!owner.update_revive())
+		return
 
-	owner.visible_message("<span class='danger'>[owner] suddenly convulses, as [owner.p_they()] stagger to [owner.p_their()] feet and gain a ravenous hunger in [owner.p_their()] eyes!</span>", "<span class='alien'>You HUNGER!</span>")
-	playsound(owner.loc, 'sound/hallucinations/far_noise.ogg', 50, 1)
+	owner.grab_ghost()
+	owner.visible_message("<span class='danger'>[owner] suddenly convulses, as [owner.p_they()][stand_up ? " stagger to [owner.p_their()] feet and" : ""] gain a ravenous hunger in [owner.p_their()] eyes!</span>", "<span class='alien'>You HUNGER!</span>")
+	playsound(owner.loc, 'sound/hallucinations/far_noise.ogg', 50, TRUE)
 	owner.do_jitter_animation(living_transformation_time)
 	owner.Stun(living_transformation_time)
```
Comment (Couls) [inline code/modules/zombie/organs.dm]: stuns for 30 ticks, or a whole minute with the new living_transformation_time variable

Comment (Fox-McCloud) [reply code/modules/zombie/organs.dm]: Yeah, oversight on my part. THanks.

**Comment by KasparoVy** [comment]
Man that was a massive pain to figure out, but quite the learning experience

**Pull request merged**